segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/ServiceStarter.java

```diff
@@ -275,14 +275,14 @@ private void attachStorage(ServiceBuilder builder) {
     static void validateConfig(ServiceBuilderConfig config) {
         long xmx = Runtime.getRuntime().maxMemory();
         long nettyDirectMem = PlatformDependent.maxDirectMemory(); //Dio.netty.maxDirectMemory
+        long totalMem = ((com.sun.management.OperatingSystemMXBean) ManagementFactory.getOperatingSystemMXBean()).getTotalPhysicalMemorySize(); 
         long maxDirectMemorySize = Long.parseLong(ManagementFactory.getPlatformMXBean(HotSpotDiagnosticMXBean.class)
                                                                    .getVMOption("MaxDirectMemorySize").getValue());
         maxDirectMemorySize = (maxDirectMemorySize == 0) ? xmx : maxDirectMemorySize;
         long cacheSize = config.getConfig(ServiceConfig::builder).getCachePolicy().getMaxSize();
-        log.info("MaxDirectMemorySize is {}, Cache size is {} and Netty DM is {}", maxDirectMemorySize, cacheSize, nettyDirectMem);
+        log.info("MaxDirectMemorySize is {}, TotalMem is {}, Cache size is {} and Netty DM is {}", maxDirectMemorySize, totalMem, cacheSize, nettyDirectMem);
```

Comment:
For better readability, as this is a log message, could you change the message to `Total Memory` instead of `TotalMem`? Also, I think it makes sense to put both the log message and `totalMem` as a first parameter in the log output instead of being second, as all the other parameters are in function of the total memory available in the instance.